storybook/stories/index.js

```diff
@@ -120,45 +121,18 @@ storiesOf('Components/Inputs', module)
   })
   .addDecorator((getStory) => <CenterView>{getStory()}</CenterView>)
   .add('Input', () => (
-    <React.Fragment>
-      <Text>Usual Input</Text>
-      <Input />
-      <Text>Error Input</Text>
-      <Input error={true} />
-      <Text>Success Input</Text>
-      <Input error={false} />
-    </React.Fragment>
+    <StoryWrapper>
+      <Input style={{width: '100%'}} placeholder='Default Input' />
```

Comment:
Why we need to specify inline style for width here? Width 100% must be default in lnput component.
styles here may be used for margins or paddings.